_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Everyone keeps their position _in the queue_. The utopia is here.
 
-This was made possible by rewriting the code so that all players run on one common service. When you switch from one player to another, this service remains running, preserving the stream data, and the new player just picks up from where the old one left off. Everything remains the same, including the queue. Ah, serendipity.
+An equally complicated and cumbersome code rewrite lets all views use one common service. Switch from one player to another, and the service is at your service preserving the stream data, changing only the view. If anything your blood-pressure could drop _too low_. Ah, serendipity.
 
 ### Video details page
 
-A more unified experience is not the only feature this change brought you. There is one which is immediately noticeable once you open a video in the new version: the video details page. Earlier, when you opened the video details, you were shown a thumbnail, video description, comments, etc. Tapping the thumbnail would open the player in full-screen, hiding the rest of these elements. Now, when you're in portrait mode, the video plays right in the video details page - just like YouTube! There is even a new 'Autoplay' setting which plays the video automatically when you open video details - also just like YouTube! This setting will be turned on by default for Wi-Fi. Of course, you can turn it off if you prefer the behaviour of the previous versions.
+All is one. If you open the video details page you probably think there is going to be a thumbnail, video description, comments, etc.
+Tapping the thumbnail would open the player in full-screen, hiding the rest of these elements. That is no way to live.
+Imagine—if in portrait mode, the video plays right in the video details page. Raise your hands and say "the day has come". Believe.
+NewPipe now comes equipped with a setting for 'Autoplay'. It can be enjoyed responsibly for automatic video play upon opening video details.
+Such lavishness is of course off by default for mobile data, but works out of the box provided Wi-Fi compatible equipment is within reach.
 
 ### Video orientation
 
-Previously, the main video player would always play videos in full-screen regardless of your selected orientation. This was fine for landscape videos in landscape mode and portrait videos in portrait mode, but it resulted in a lot of wasted space if the video's orientation didn't match the player's. No more! There is now a full-screen button which replaces the orientation switch button, and it is _smart_. It will detect if the video is landscape or portrait, and will ensure the player is in the correct orientation when you enter full-screen! How's _that_ for convenience and usability? Not only that, but if auto-rotate is enabled, and the video is a landscape one, the button will disappear because you don't need it! (In portrait mode, you still need the button to go into full-screen from the video details.)
```

Comment:
"Previously" is superfluous. This was the hardest part to read. It requires a lot of mental adaptation, and it is mechanical. The strings don't use "enabled". Double "that".
>Not only that, but if auto-rotate is enabled, and the video is a landscape one, the button will disappear because you don't need it!

This sentence is horrendous. Four clauses to the point, which is at the end of all that.

Reply:
> "Previously" is superfluous.

"Earlier"?

> it is mechanical

What does "mechanical" mean here? You've used this word earlier as well, but I don't get it from the context.

> The strings don't use "enabled".

Strings? What strings?

> Double "that".

Where?

> Four clauses to the point

Three, actually. And considering there are two conditions and one result, that is unavoidable. Your replacement, incidentally, _also_ contains three clauses, so I'm not sure how it is better?

> If you have auto-rotate on, the button vanishes for regular videos, but ready to go from vertical video details, to full vertical video on the vertical.

Reply:
I can't bring myself to read that again so I understand it, but the ordering, the lack of repetition. ("will" twice), the many sentences, the length. It is a mess. It is still a mess, but it can now be disregarded somewhat.